python/repositories.bzl

```diff
@@ -192,6 +193,9 @@ python_repository = repository_rule(
             doc = "The SHA256 integrity hash for the Python interpreter tarball.",
             mandatory = True,
         ),
+        "strip_prefix": attr.string(
+            doc = "A directory prefix to strip from the extracted files.",
```

Comment:
either add `mandatory=True` or add the `default="python"` ?

I don't think a default of empty string is ever useful when actually extracting?

Reply:
It could be if someone packaged a toolchain that had everything in the root directory. I think setting `mandatory=True` is incorrect and `default="python"` forces a precedent which doesn't need to be true. An empty string would be equivalent to no strip prefix.

All that said though, since all the toolchains currently come from https://github.com/indygreg/python-build-standalone, I could set `default="python"` if you'd like but still advise against it. Your thoughts?

Reply:
I'd like one of mandatory (you can pass empty string if that's the right value) or default="python"

Reply:
The attribute is now mandatory